core-library/src/main/java/org/silverpeas/core/admin/user/UserIndexation.java

```diff
@@ -25,6 +25,10 @@
 
 import org.silverpeas.core.admin.user.model.UserDetail;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.silverpeas.core.admin.user.model.UserDetail'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)